core/common/src/tile/TileIO.ts

```diff
@@ -62,6 +62,7 @@ export enum TileReadStatus {
   InvalidFeatureTable,
   NewerMajorVersion,
   Canceled,
+  UnsupportedMeshoptCompression,
```

Comment:
@pmconne I added a dedicated error in case meshopt failed to load AND compressed tiles are provided, yea of nay ?

Reply:
Nay, nobody checks the detailed statuses and if we find ourselves in this situation then we messed up.

Reply:
the error code was removed